packages/jest-snapshot/src/index.ts

```diff
@@ -4,17 +4,15 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {HasteFS} from 'types/HasteMap';
-import type {MatcherState} from 'types/Matchers';
-import type {Path, SnapshotUpdateState} from 'types/Config';
-import type {SnapshotResolver} from 'types/SnapshotResolver';
-
 import fs from 'fs';
+import {Config, MatcherState} from '@jest/types';
+import {FS as HasteFS} from 'jest-haste-map'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
should be peer dep and dev dep, I'd say. Since the cleanup functions receive them, it's part of this module's public API